website/docs/dev/ide-setup.md

```diff
@@ -105,10 +105,10 @@ It is recommended to automatically format your code by applying the following se
 
 For earlier IntelliJ IDEA versions:
 
-6. Go to "Settings" → "Other Settings" → "Save Actions".
-7. Under "General", enable your preferred settings for when to format the code, e.g. "Activate save actions on save".
-8. Under "Formatting Actions", select "Optimize imports" and "Reformat file".
-9. Under "File Path Inclusions", add an entry for `.*\.java` to avoid formatting other file types.
+1. Go to "Settings" → "Other Settings" → "Save Actions".
```

Comment:
https://github.com/alibaba/fluss/pull/234/files#diff-a196e06cbb74e690aab3e19dcf834dfddb0ce960a80ec2d50386600558890c75L102
should be change to `4`, and then the seq number also should be change to `4`.....